src/ompl/geometric/planners/rrt/src/RRTstar.cpp

```diff
@@ -207,9 +261,8 @@ ompl::base::PlannerStatus ompl::geometric::RRTstar::solve(const base::PlannerTer
             goal_s->sampleGoal(rstate);
         else
         {
-            sampler_->sampleUniform(rstate);
-
-            if (prune_ && opt_->isCostBetterThan(bestCost_, costToGo(rmotion)))
+            // Attempt to generate a sample, if we fail (e.g., too many rejection attempts), skip the remainder of this loop and return to try again
+            if (!sampleUniform(rstate))
```

Comment:
The rejection sampling that was being performed by the line `opt_->isCostBetterThan(bestCost_, costToGo(rmotion))` is now being done by the default rejection sampler that was recently introduced.

`sampleUniform` looks after calling the appropriate sampler and returns false if the sampler times out.